### PR TITLE
Konstantinos / Task - Refactor contact-us page [Translation Key Extraction]

### PR DESCRIPTION
**Pull request by konstantinosG-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [x] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Review by mitra-deriv — APPROVED** [review]

File: src/pages/contact_us/index.tsx [hunk 1]
```diff
@@ -39,19 +46,17 @@ const ContactUs = () => {
     return (
         <Layout>
             <SEO
-                title={localize('Contact Us | Online Trading | Deriv.com')}
-                description={localize(
-                    'Have any query or need assistance, check out the details to call our international help desk and our email ID. Feel free to contact us anytime!',
-                )}
+                title={localize(title_text)}
+                description={localize(query_text)}
```
Comment (niloofar-deriv) [inline src/pages/contact_us/index.tsx:50]:
```suggestion
                description={localize(description_text)}
```

File: src/pages/contact_us/index.tsx [hunk 2]
```diff
@@ -39,19 +46,17 @@ const ContactUs = () => {
...
                 has_organization_schema
             />
             <SectionContainer tablet={{ p: '40px 0' }}>
                 <HeroWrapper fd="column">
                     <Header align="center" as="h1" type="display-title">
-                        {localize('Contact us')}
+                        {localize(contact_us_text)}
```
Comment (niloofar-deriv) [inline src/pages/contact_us/index.tsx:56]:
```suggestion
                        <Localize translat_text={contact_us_text} />
```

File: src/pages/contact_us/index.tsx [hunk 3]
```diff
@@ -39,19 +46,17 @@ const ContactUs = () => {
...
                     </Header>
                     <StyledText align="center" size="var(--text-size-sm)">
-                        {localize("Got questions? Here's how to get answers.")}
+                        {localize(questions_text)}
```
Comment (niloofar-deriv) [inline src/pages/contact_us/index.tsx:59]:
```suggestion
                        <Localize translate_text={questions_text} />
```

Comment (mitra-deriv) [reply src/pages/contact_us/index.tsx:59]: @niloofar-deriv what is the benefit of this approach over the old one ? we should use it like this everywhere ? 🙏

Comment (niloofar-deriv) [reply src/pages/contact_us/index.tsx:59]: As I mentioned in the parent card this is really important, our components need to be attached to react life cycle, so basically using `localize` in most of places are not quite ok.

**Review by niloofar-deriv — APPROVED** [review]
LGTM :+1:

**Review by habib-deriv — APPROVED** [review]
LGTM

**Pull request closed without merging**